test/e2e/main_test.go

```diff
@@ -409,3 +409,6 @@ func testManualMeteringInstall(
 	err = deployerCtx.Teardown(deployerCtx.Deployer.Uninstall)
 	assert.NoError(t, err, "capturing logs and uninstalling metering should produce no error")
 }
+
+func TestSharedStorage(t *testing.T) {
+	testInstallConfigs := []struct {
```

Comment:
We would most likely add this type of test as an additional test table index in the top-level `TestManualMeteringInstall` test function.